PKGBUILD

```diff
@@ -9,106 +10,54 @@
-"${pkgname}-${pkgver}-opencl-lib.tar.gz::https://skia.googlesource.com/external/github.com/GPUOpen-Tools/\
-common-lib-amd-APPSDK-3.0/+archive/4e6d30e406d2e5a65e1d65e404fe6df5f772a32b.tar.gz"
-"${pkgname}-${pkgver}-opencl-registry.tar.gz::https://skia.googlesource.com/external/github.com/\
-KhronosGroup/OpenCL-Registry/+archive/932ed55c85f887041291cef8019e54280c033c35.tar.gz"
-"${pkgname}-${pkgver}-opengl-registry.tar.gz::https://skia.googlesource.com/external/github.com/\
-KhronosGroup/OpenGL-Registry/+archive/14b80ebeab022b2c78f84a573f01028c96075553.tar.gz"
-"${pkgname}-${pkgver}-piex.tar.gz::https://android.googlesource.com/platform/external/piex.git/+archive/\
-bb217acdca1cc0c16b704669dd6f91a1b509c406.tar.gz"
-"${pkgname}-${pkgver}-sdl.tar.gz::https://skia.googlesource.com/third_party/sdl/+archive/\
-5d7cfcca344034aff9327f77fc181ae3754e7a90.tar.gz"
-"${pkgname}-${pkgver}-sfntly.tar.gz::https://chromium.googlesource.com/external/github.com/googlei18n/\
-sfntly.git/+archive/b55ff303ea2f9e26702b514cf6a3196a2e3e2974.tar.gz"
-"${pkgname}-${pkgver}-spirv-cross.tar.gz::https://chromium.googlesource.com/external/github.com/\
-KhronosGroup/SPIRV-Cross/+archive/53ab2144b90abede33be5161aec5dfc94ddc3caf.tar.gz"
-"${pkgname}-${pkgver}-spirv-headers.tar.gz::https://skia.googlesource.com/external/github.com/KhronosGroup/\
-SPIRV-Headers.git/+archive/29c11140baaf9f7fdaa39a583672c556bf1795a1.tar.gz"
-"${pkgname}-${pkgver}-spirv-tools.tar.gz::https://skia.googlesource.com/external/github.com/KhronosGroup\
-/SPIRV-Tools.git/+archive/0c4feb643b89d1792b02f7cbef315e9d95633bd7.tar.gz"
-"${pkgname}-${pkgver}-swiftshader.tar.gz::https://swiftshader.googlesource.com/SwiftShader/+archive/\
-430def835f9f85d52f4a96db9b715cd9a7403c9c.tar.gz"
-"${pkgname}-${pkgver}-wuffs.tar.gz::https://skia.googlesource.com/external/github.com/google/wuffs.git\
-/+archive/4080840928c0b05a80cda0d14ac2e2615f679f1a.tar.gz"
-"${pkgname}-${pkgver}-zlib.tar.gz::https://chromium.googlesource.com/chromium/src/third_party/zlib/+archive/\
-ea3ba903faac98b64b2bf8de5e98cd97b335a474.tar.gz"
-"gn::https://chromium-gn.storage-download.googleapis.com/3523d50538357829725d4ed74b777a572ce0ac74"
-)
-_gitsrc=("${source[@]:4:29}")
-noextract=("${_gitsrc[@]%%::*}")
+depends=(# ~ Aseprite's direct dependencies ~
+         # pixman is not linked to because we use Skia instead
+         # harfbuzz is linked statically because Aseprite expects an older version
+         cmark libcurl.so libgif.so libjpeg.so zlib libpng tinyxml libfreetype.so libarchive.so
+         hicolor-icon-theme # For installing Aseprite's icons
+         # ~ Skia deps ~
+         # (Skia links dynamically to HarfBuzz, only Aseprite itself doesn't. >_<)
+         libexpat.so=1-64 libharfbuzz.so=0-64
+         # Already required by Aseprite: libjpeg-turbo libpng zlib freetype2
+         # These two are only reported by Namcap, but don't seem to be direct dependencies?
+         libfontconfig.so libxcursor)
+makedepends=(# "Meta" dependencies
```

Comment:
missing `libglvnd`

Reply:
Yeah, I noticed the run failed an got to work. I think the proper dependency is `libgl`, but [see my commit's message](https://github.com/ImperatorStorm/PKGBUILDs/pull/1/commits/60524054ec445a5a960436aa93ec8f9675162761) for why I am unsure about the runtime-ness of this dep.

Thanks for setting up the CI!

Reply:
np.